user_guide_src/source/incoming/routing.rst

```diff
@@ -56,7 +23,7 @@ If you expect a GET request, you use the ``get()`` method:
 
 .. literalinclude:: routing/001.php
 
-A route simply takes the URI path on the left, and maps it to the controller and method on the right,
+A route simply takes the URI path (``/``) on the left, and maps it to the controller and method (``Home::index``) on the right,
```

Comment:
"simply" can be removed? https://jameshfisher.com/2017/02/22/dont-use-simply/

Reply:
@samsonasik Thanks! Done.